src/index.html

```diff
@@ -11,7 +11,193 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+    >
+      <div class="wrapper">
+        <fieldset class="fields">
+          <legend>Personal information</legend>
+          <div class="field">
+            <label for="userSurname">Surname:</label>
+            <input
+              type="text"
+              name="userSurname"
+              id="userSurname"
+            >
+          </div>
+
+          <div class="field">
+            <label for="userName">Name:</label>
+            <input
+              type="text"
+              name="userName"
+              id="userName"
+            >
+          </div>
+
+          <div class="field">
+            <label for="age">How old are You?</label>
+            <input
+              type="number"
+              name="age"
+              required
+              min="1"
+              max="100"
+              value="12"
+                          id="age"
+            >
```

Comment:
Why do we need so many spaces?

```suggestion
              id="age"
            >
```